agents/acu/acu_agent.py

```diff
@@ -379,106 +567,124 @@ def datagramReceived(self, data, src_addr):
-                    elevation_corrected = d[3]
-                    elevation_raw = d[5]
-                    self.data['broadcast'] = {'Time': data_ctime,
-                                              'Azimuth_Corrected': azimuth_corrected,
-                                              'Azimuth_Raw': azimuth_raw,
-                                              'Elevation_Corrected': elevation_corrected,
-                                              'Elevation_Raw': elevation_raw,
-                                              }
+                    self.data['broadcast']['Time'] = data_ctime
+                    for i in range(2, len(d)):
+                        self.data['broadcast'][fields[i].replace(' ', '_')] = d[i]
                     acu_udp_stream = {'timestamp': self.data['broadcast']['Time'],
-                                      'block_name': 'ACU_position',
+                                      'block_name': 'ACU_broadcast',
                                       'data': self.data['broadcast']
                                       }
+                    #print(acu_udp_stream)
                     self.agent.publish_to_feed('acu_udp_stream',
-                                               acu_udp_stream)
+                                               acu_udp_stream, from_reactor=True)
             else:
                 yield dsleep(1)
             yield dsleep(0.005)
 
         handler.stopListening()
-        self.set_job_done('broadcast')
+        self._set_job_done('broadcast')
         return True, 'Acquisition exited cleanly.'
 
     @inlineCallbacks
-    def go_to(self, session, params=None):
-        """ TASK "go_to"
+    def go_to(self, session, params):
+        """go_to(az=None, el=None, wait=1)
 
-        Moves the telescope to a particular point (azimuth, elevation)
-        in Preset mode. When motion has ended and the telescope reaches
-        the preset point, it returns to Stop mode and ends.
+        **Task** - Move the telescope to a particular point (azimuth,
+        elevation) in Preset mode. When motion has ended and the telescope
+        reaches the preset point, it returns to Stop mode and ends.
 
-        Params:
+        Parameters:
             az (float): destination angle for the azimuthal axis
             el (float): destination angle for the elevation axis
             wait (float): amount of time to wait for motion to end
+
         """
-        ok, msg = self.try_set_job('control')
+        ok, msg = self._try_set_job('control')
         if not ok:
             return ok, msg
         az = params.get('az')
         el = params.get('el')
+        if az <= self.motion_limits['azimuth']['lower'] or az >= self.motion_limits['azimuth']['upper']:
+            return False, 'Azimuth location out of range!'
+        if el <= self.motion_limits['elevation']['lower'] or el >= self.motion_limits['elevation']['upper']:
+            return False, 'Elevation location out of range!'
```

Comment:
A couple notes on this implementation:
- `self` isn't accessible in the decorator. ["Validation in the Op function"](https://ocs.readthedocs.io/en/develop/developer/agents.html#validation-in-the-op-function) needed to be used here
- If the param decorator is used to describe one parameter it must be used to describe all parameters, otherwise parameters not described with a decorator are considered unknown and will be rejected

Since I'm running the end to end test setup on this branch and was trying to fix the issues resolved in [65d54d1](https://github.com/simonsobs/socs/pull/294/commits/65d54d1be5121c1c7de12950ef3146a06f7ee033) I went ahead and made the changes needed in [96a32c1](https://github.com/simonsobs/socs/pull/294/commits/96a32c190e41876d37e87dfe7ca576c98eb6fbce).